x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -245,9 +299,14 @@ const allowlistEventFields: AllowlistFields = {
   '@timestamp': true,
   agent: true,
   Endpoint: true,
+  Ransomware: true,
+  data_stream: true,
   ecs: true,
   elastic: true,
   event: true,
+  rule: {
```

Comment:
Can you push through a doc with this field, and with the PR for the fix to push the `event.code` field? This otherwise looks 💯 but I'd love to see this upstream too :)

Reply:
Sure thing. I will ping the Endpoint App team now to approve: https://github.com/elastic/kibana/pull/85109
I will ping you the document offline due to it being in a private cluster 😃